terraform/workspace-variables/pentest.tfvars

```diff
@@ -0,0 +1,36 @@
+# Platform
+region                      = "eu-west-2"
+environment                 = "pentest"
+app_environment             = "pentest"
+parameter_store_environment = "qa"
+
+# CloudFront
+distribution_list = {
+  "tvsqa" = {
```

Comment:
This should be called tvspentest to avoid confusion

Reply:
done